docs/usage/assets/background-tasks.md

```diff
@@ -0,0 +1,26 @@
+# Background tasks
```

Comment:
May be add a screenshot that shows where we see the difference between Knowledge/User on the screen ?

Reply:
right! I'm going to add a 'scope' chip near the background tasks listing

Reply:
done !